.github/workflows/cont_integration.yml

```diff
@@ -12,19 +12,17 @@ jobs:
         rust:
           - version: stable
             clippy: true
-          - version: 1.57.0 # MSRV
+          - version: "1.57.0" # MSRV
```

Comment:
Yaml is complicated, things like `1.60.0` will be rounded to `1.6`.
As a best practice use strings.
Also check [YAML Hell](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell)